src/core.js

```diff
@@ -259,7 +259,25 @@ $.extend( $.validator, {
 			}
 		},
 		onkeyup: function( element, event ) {
-			if ( event.which === 9 && this.elementValue( element ) === "" ) {
+			// Avoid revalidate the field when pressing one of the following keys
+			// Shift       => 16
```

Comment:
The  `ctrl` key trigger the `keyup` event when used with another key as combination ( `ctrl + c`, `ctrl + v`, ...)
Do we have to check for it also ?